.env

```diff
@@ -0,0 +1,2 @@
+PORT=3000
+DB_ENV=testing
```

Comment:
remember this should be `development` just to pull double-duty in case you have this API deployed to a service like heroku--if you use the cross-env variable (using the cross-env library) like Luis did during his demo yesterday and include `cross-env DB_ENV=testing` in your test script command, it will override this variable to be testing, otherwise you still want to be able to run and modify your non-test database (development)